src/elements/content-sidebar/versions/VersionsSidebarContainer.js

```diff
@@ -95,11 +95,6 @@ class VersionsSidebarContainer extends React.Component<Props, State> {
         }
     }
 
-    componentWillUnmount() {
```

Comment:
Hey so are there any possible side effects of removing this code completely.  If we can remove it then it looks like it may have been unnecessary code in the first place.

Reply:
It also means that switching to other sidebars, like "Details" or "Metadata", will not revert to the current version preview. The only mechanism to do so will be the button in the header.

Have we heard back from the relevant stakeholders that this change is acceptable even on larger screen sizes?

Reply:
Have approval for the version behavior change, ie to show the previous version after closing the right sidebar on large screens. However, let me circle back after checking about the "Details" and "Metadata" experiences.

Reply:
@greathmaster, understood. I would think the Next/Previous buttons to navigate forward and backward through previews in a folder may also be affected.